website/docs/feature-flag-tutorials/react/examples.md

```diff
@@ -0,0 +1,627 @@
+
+In Unleash Enterprise, we have a change request feature in your project settings to manage your feature flag approvals. When someone working in a project needs to update the status of a flag or strategy, they can follow our approval workflow to ensure that one or more team members review the change request.
+
+
+![Project settings](/img/react-ex-project-settings.png)
+
+
+We have several statuses that indicate the stages a feature flag could be in as it progresses through the workflow. This facilitates collaboration on teams while reducing risk in environments like production. For larger scale change management, you can ensure the correct updates are made while having full visibility into who made the request for change and when.
+
+
+![Change requests](/img/react-ex-change-requests.png)
+
+
+Learn more about [how to configure your change requests](/reference/change-requests) and our [project collaboration mode](/reference/project-collaboration-mode).
+
+
+## Flag Automation & Workflow Integration for React Apps
+
+An advanced use case for leveraging feature flags at scale is flag automation in your development workflow. Many organizations use tools like Jira for managing projects and tracking releases across teams. [Our Jira integration](/reference/integrations/jira-server-plugin-installation) helps to manage feature flag lifecycles associated with your projects.
+
+It’s common for teams to have a development phase, QA/testing and then a release to production. Let’s say the changes we’ve made in our React tutorial project need to go through a typical development workflow. The [Unleash Jira plugin](https://marketplace.atlassian.com/apps/1227377/unleash-for-jira?tab=overview&hosting=datacenter) can connect to your Jira server or cloud to create feature flags automatically during the project creation phase. As your code progresses through development and Jira tickets are updated, the relevant flag can turn on in a development environment. The next stage could be Canary deployments for testing code quality in subsequent environments to certain groups, like a QA team or beta users. The flag could be automatically turned on in QA and rollout and/or rollout to target audiences in production.
+
+Here’s how this can be done via our API:
+
+
+
+1. Enable a flag.
+
+```
+curl -L -X POST '<your-unleash-url>/api/admin/projects/:projectId/features/:featureName/environments/:environment/on' \
+-H 'Accept: application/json' \
+-H 'Authorization: <API_KEY_VALUE>'
+```
+
+Review our [API docs on flag enablement](/reference/api/unleash/toggle-feature-environment-on).
```

Comment:
In the rendering, this appears to be a separate paragraph and isn't nested under this list point (notice the indentation):

<img width="382" alt="image" src="https://github.com/Unleash/unleash/assets/17786332/976b8a1b-8baf-4219-89a5-e30c1dce4d3b">

I'm not entirely sure what the syntax is to nest it properly, but I know it's possible. Maybe it's some extra indentation?

This applies to the next few review suggestions as well.